src/data/queries/tx.ts

```diff
@@ -50,6 +51,24 @@ export const useTxInfo = ({ txhash, queryKeys, chainID }: LatestTx) => {
   )
 }
 
+export const useOsmosisGas = () => {
+  const { networks } = useNetworks()
+
+  return useQuery(
+    [queryKey.tx.osmosisGas],
+    async () => {
+      const { data } = await axios.get(
+        "osmosis/txfees/v1beta1/cur_eip_base_fee",
+        {
+          baseURL: networks["mainnet"]["osmosis-1"].lcd, // hard set for now.
+        }
+      )
+      return data
+    },
+    { ...RefetchOptions.INFINITY }
```

Comment:
We might want to use:
```ts
{ 
  ...RefetchOptions.DEFAULT, 
  staleTime: 60 * 1000 // cache data for 1 min
}
```
since the gas price changes often.